syft/workers/base.py

```diff
@@ -843,6 +850,9 @@ def generate_triple(
         c_shared = c.share(*locations, field=field, crypto_provider=self).child
         return a_shared, b_shared, c_shared
 
+    def save_promise(self):
```

Comment:
Why does this method that does nothing exist?

Reply:
removed